### DISCUSSION TITLE
is smplayer project dead?

### DISCUSSION THREAD

**Discussion by premier69** [smplayer-dev/smplayer#9] [Q&A] [answered]
someone on a youtube comment just said so, is it true?

**Comment by smplayer-dev** [comment, marked as answer]
No.

**Reply by dandv** [reply]
Has the whole forum, with 10+ YEARS of discussions, been killed? It now redirects here, but Google search results are rotten links.

![image](https://user-images.githubusercontent.com/33569/112059501-00866280-8b00-11eb-9f84-ce2b26e5472f.png)

**Reply by Cootmaster** [reply]
http://old-forum.smplayer.info/  old forum is here, but you cant POST  ANYTHING/NEW

**Reply by smplayer-dev** [reply]
Ok, http://forum.smplayer.info is back, but it's still broken.

**Reply by Cootmaster** [reply]
ummm  ya its dead  ..  now the link on the  Support forum: http://forum.smplayer.info
  interface  takes me to old forun   but i cant do anything  , last week it sent me to github now back to forum .  how am i supposed to come here if the   interface sends me to forum?
